<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NounDialog</class>
 <widget class="QDialog" name="NounDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="enEttLabel">
     <property name="text">
      <string>en/ett</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="bestSingLabel">
     <property name="text">
      <string>best채md singular</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="bestSingField"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="obestSingLabel">
     <property name="text">
      <string>obest채md singular</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="obestSingField"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="bestPlurLabel">
     <property name="text">
      <string>best채md plural</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="bestPlurField"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="obestPlurLabel">
     <property name="text">
      <string>obest채md plural</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="obestPlurField"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="enEttField">
     <item>
      <property name="text">
       <string>en</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ett</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="engTransField"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="engTransLabel">
     <property name="text">
      <string>english translation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NounDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NounDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
